_includes/icml/about.icml.html

```diff
@@ -1,3 +1,4 @@
+<br/> &nbsp; &gt; <a class="project-link" href="/icml-2019-es">Español</a>
```

Comment:
this should be added to the nav bar rather than linked on the main page.

Reply:
Yes, I agree with this. I can have the update on Friday. If you want to accept it with the button for now, that would get the urgent things forward. But I won't be able to make much progress until Friday (sad face emoji here).